packages/clay-core/stories/SidePanel.stories.tsx

```diff
@@ -94,14 +120,83 @@ export const Default = () => {
 					onOpenChange={setOpen}
 					open={open}
 				>
-					<SidePanel.Header>
-						<SidePanel.Title>Right Panel</SidePanel.Title>
+					<SidePanel.Header className="asdfadsf" sticky>
```

Comment:
This looks like an unintended added className

Reply:
oops thanks for getting this